### ISSUE TITLE
Is there a built-in for disabling shadows?

### ISSUE THREAD

**Issue by adelriosantiago** [papercss/papercss#234]
Is there a built-in for disabling shadows? Example `<div class="sm-3 col no-shadow">No shadow</div>` so that this particular item will not have a shadow?

**Comment by rhyneav** [comment]
There is not, but shadows aren't enabled by default. Are you looking for an override when using `child-shadows`?

**Comment by adelriosantiago** [comment]
Yes, exactly. So that I can select which ones should **not** have a shadow under a `child-shadows` parent.

**Comment by rhyneav** [comment]
Ah got it, that makes sense! A PR would be welcome here, probably with the class named as `shadow-none` to keep consistent with the existing names. I'll update this issue to reflect that for if you or someone else wants to pick this one up!

**Comment by toshihidetagami** [comment]
Hi, this is a cool project. Can I take up this issue?

**Comment by rhyneav** [comment]
Please do @toshihidetagami! Let me know if you need any additional information!
